### ISSUE TITLE
Is it possible to target nested object properties?

### ISSUE THREAD

**Issue by kevinschaich** [juliangarnier/anime#842]
e.g. I have:

```javascript
const myObject = {
    a: {
        b: {
            c: 1000
        }
    }
}
```

Can I somehow tell anime that I want 'c' to change? via dot notation `a.b.c` or similar?

**Comment by different55** [comment]
Yeah, you'd use dot notation. Not somewhere where I can cough up a mini example but check out the [docs](https://animejs.com/documentation/). They show how to target an arbitrary JavaScript object and animate its properties. But I believe you'd target a.b and then set c to whatever values/keyframes.

**Comment by kevinschaich** [comment]
For right now since we couldn't find any example code, I decided to just "flatten" my object to make the top-level keys strings:

```typescript
const myObject = {
    'a.b.c': 1000
}
```

and in anime:

```typescript
anime({
    targets: myObject,
    'a.b.c': 2000,
    ...
})
```

Let me know if anyone has a better way!

**Comment by PAEz** [comment]
Never rEaLlY used this library, but just looking at what is said here then wouldnt it just be.....
```
anime({
    targets: myObject.a.b,
    'c': 2000,
    ...
})
```

**Comment by juliangarnier** [comment]
Duplicate of #731
